### PR TITLE
Enable and fix test, fix error handling and add new packages to upgrade.sh.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/integration_test/run_tests.dart [hunk 1]
```diff
@@ -28,10 +28,10 @@ void main(List<String> args) async {
   Exception? exception;
   final chromedriver = ChromeDriver();
 
-  try {
-    // Start chrome driver before running the flutter integration test.
-    await chromedriver.start();
+  // Start chrome driver before running the flutter integration test.
+  await chromedriver.start();
```
Comment (kenzieschmoll) [inline packages/devtools_app/integration_test/run_tests.dart:32]: why is this moved out of the try block? if an exception is thrown we need to catch it.

Comment (polina-c) [reply packages/devtools_app/integration_test/run_tests.dart:32]: We rethrow the exception anyway.
But if start fails, finally also fails because late _process in ChromeDriver is not initialized.

Comment (kenzieschmoll) [reply packages/devtools_app/integration_test/run_tests.dart:32]: Let's just make `_process` nullable in the `ChromeDriver` class, and put this back into the try block. We only rethrow on line 62 for exceptions caught in the try block.

Comment (polina-c) [reply packages/devtools_app/integration_test/run_tests.dart:32]: What is the point to save exception in order to rethrow it?
I would just remove `catch` from here.

File: packages/devtools_app/integration_test/run_tests.dart [hunk 2]
```diff
@@ -52,15 +51,9 @@ void main(List<String> args) async {
         await _runTest(newArgsWithTarget);
       }
     }
-  } on Exception catch (e) {
-    exception = e;
```
Comment (kenzieschmoll) [inline packages/devtools_app/integration_test/run_tests.dart:56]: please don't remove all this exception handling.

**Review by kenzieschmoll — APPROVED** [review]

File: packages/devtools_app/integration_test/run_tests.dart (hunk 2 above)
Comment (polina-c) [reply packages/devtools_app/integration_test/run_tests.dart:56]: We discussed that this exception handling does not change behavior.
